docs/source/getting_started.ipynb

```diff
@@ -0,0 +1,614 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25083f32",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tmgr.add_pilots(pilot)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "13e7458e",
+   "metadata": {},
+   "source": [
+    "### Describing the application tasks\n",
+    "\n",
+    "In this example, we want to run simple tasks but that require different number of CPU cores and that run for a variable amount of time. Thus, we use the executable [`radical-pilot-hello.sh`](https://github.com/radical-cybertools/radical.pilot/blob/devel/bin/radical-pilot-hello.sh) we crafted to occupy a configurable amount of resources for a configurable amount of time. \n",
+    "\n",
+    "Each task is an instance of `radical.pilot.TaskDescription` with some properties defined (for a complete list of task properties see the [TaskDescription API](apidoc.rst#taskdescription):\n",
+    "* `executable`: the name of the executable we want to launch with the task\n",
+    "* `arguments`: the arguments to pass to `executable`. In this case, the number of seconds it needs to run for.\n",
+    "* `ranks`: this is the number of nodes on which the task should run. Here it is set to 1 as we are running all our tasks on our local computer. See [Describing tasks in RADICAL-Pilot](tutorials/describing_tasks.ipynb) and the details about executing tasks that use the message passing interface (MPI).\n",
+    "* `cores_per_rank`: the amount of cores that each (rank of the) task utilizes. In our case, each task will randomly use either 1 or 2 cores or the 4 we have requested.\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "__Warning:__ Executing MPI tasks (i.e., one with `radical.pilot.TaskDescription.ranks > 1) requires for an MPI implementation to be available on the machine on which you will run the task. That is usually taken care of by the system administrator but if you are managing your own cluster, you will have to install and make avaialable one of the many MPI distributions available for GNU/Linux. \n",
+    "\n",
+    "</div>\n",
+    "\n",
+    "We run 10 tasks that should be enough to see both concurrent and sequential executions on the amount of resources we requested, but not enough to clog the example.\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "    \n",
+    "__Note:__ We use the reporter to produce a progress bar while we loop over the task descriptions\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ba782cd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import random\n",
+    "\n",
+    "n = 10\n",
+    "\n",
+    "report.progress_tgt(n, label='create')\n",
+    "tds = list()\n",
+    "for i in range(n):\n",
+    "\n",
+    "    td = rp.TaskDescription()\n",
+    "    td.executable     = '%s/radical-pilot-hello.sh' % os.getcwd()\n",
```

Comment:
`'%s/radical-pilot-hello.sh' % os.getcwd()` -> `'radical-pilot-hello.sh'` (since this script is within `/bin` directory in VE)